pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -7,16 +7,13 @@
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "tracker-miners";
-  version = "2.0.4";
+  version = "2.0.5";
 
   src = fetchurl {
     url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${name}.tar.xz";
-    sha256 = "0mp9m2waii583sjgr61m1ni6py6dry11r0rzidgvw1g4cxhn89j6";
+    sha256 = "00k8nb8dbkjnqjk12gcs5n2cm6yny553qildsm5b2c8nfs1w16b4";
   };
 
-  # https://github.com/NixOS/nixpkgs/issues/39547
-  LIBRARY_PATH = stdenv.lib.makeLibraryPath [ giflib ];
```

Comment:
Apparently, this was an error in the `meson.build` https://gitlab.gnome.org/GNOME/tracker-miners/commits/master